Core/Security/Locations.py

```diff
@@ -74,11 +74,36 @@ def getCAsDefaultLocation():
 
 
 def getHostCertificateAndKeyLocation(specificLocation=None):
-  """ Retrieve the host certificate files location
+  """ Retrieve the host certificate files location.
+
+      Lookup order:
+
+      * ``specificLocation`` (probably broken, don't use it)
```

Comment:
? Why "probably broken"?

Reply:
because it is used in some places as if it was a physical path while in fact it is a CS path, and also because it is used to get both the key and the cert location, but it returns the same path for both. Which we know does not happen. 
It is used only in a very convoluted way in the `Bundle` service. Looking at the logs, this comes from factorization done by Zoltan, but I am not sure he really understood what was happening there